### PR TITLE
Add nymph and dionea seeds

### PR DESCRIPTION
**Pull request by Marya-The-Slime**
## About the PR 

**Media**
Скриншот 27 12 22_10 35 14
Скриншот 27 12 22_10 35 46
Скриншот 27 12 22_10 38 32
Скриншот 27 12 22_10 34 52

- [ I have added screenshots/videos to this PR showcasing its changes ingame, ]
**Changelog**
:cl: Marya
- add: Added nymphs
- add: Added dionea spores
- add: dionea spores to Exotic Botany crate in cargo

### CODE REVIEW

**Comment by mirrorcult** [comment]
what exactly is the purpose of these?

**Comment by Jackrost** [comment]
> what exactly is the purpose of these?

In SS13 botanics can create mobs for ghosts with those seeds

File: Resources/Textures/Mobs/Animals/nymph.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "#DionaNotAWood",
```
Comment (EmoGarbage404) [inline Resources/Textures/Mobs/Animals/nymph.rsi/meta.json:7]: if these are from bay, then you need to use the license that used on bay. A look on the repo says that the assets are licensed under the `CC-BY-SA-3.0` license.

File: Resources/Textures/Objects/Specific/Hydroponics/nymph_seed.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "version": 1,
+  "license": "#DionaNotAWood",
```
Comment (EmoGarbage404) [inline Resources/Textures/Objects/Specific/Hydroponics/nymph_seed.rsi/meta.json:3]: same as before

File: Resources/Textures/Objects/Specific/Hydroponics/nymph_seed.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "copyright": "Taken from baystation12",
```
Comment (EmoGarbage404) [inline Resources/Textures/Objects/Specific/Hydroponics/nymph_seed.rsi/meta.json:4]: link to the repo, please

File: Resources/Prototypes/Entities/Objects/Specific/Hydroponics/nymph_seed.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  id: NymphSeed
+  name: pack of dionea spores
+  noun: seeds-noun-seeds
+  displayName: pack of dionea spores
+  plantRsi: Objects/Specific/Hydroponics/nymph_seed.rsi
+  productPrototypes:
+    - MobNymph
+  lifespan: 50
+  maturation: 8
+  production: 3
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Specific/Hydroponics/nymph_seed.yml:22]: The maturation value should be higher and the production and yield values should be lower. As it stands, you can make an absurd amount of ghost roles using these.

File: Resources/Prototypes/Entities/Objects/Specific/Hydroponics/nymph_seed.yml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  name: pack of dionea spores
+  id: NymphSeed
+  components:
+    - type: Seed
+      seedId: NymphSeed
+    - type: Sprite
+      sprite: Objects/Specific/Hydroponics/nymph_seed.rsi
+
+
+- type: seed
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Specific/Hydroponics/nymph_seed.yml:12]: this doesn't need to be it's own file. Put the seed entity with the other ones and the seed prototype with the rest of the seed prototypes.

File: Resources/Prototypes/Entities/Mobs/NPCs/nymph.yml [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+        - ReagentId: Water
+          Quantity: 50
+  - type: Butcherable
+    spawned:
+    - id: NymphSeed
+      amount: 3
+  - type: Tag
+    tags:
+    - Trash
+    - CannotSuicide
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Mobs/NPCs/nymph.yml:71]: the `CannotSuicide` tag isn't really necessary here.

File: Resources/Prototypes/Entities/Mobs/NPCs/nymph.yml [hunk 7]
```diff
@@ -0,0 +1,99 @@
+- type: entity
+  name: Nymph
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Mobs/NPCs/nymph.yml:2]: nitpick but please structure your yaml according to the [conventions](https://docs.spacestation14.io/en/getting-started/conventions).
image

File: Resources/Locale/en-US/npc/nymph.ftl [hunk 8]
```diff
@@ -0,0 +1,3 @@
+ent-MobNymph = Nymph
```
Comment (EmoGarbage404) [inline Resources/Locale/en-US/npc/nymph.ftl:1]: we don't localize entity names on the main branch.

File: Resources/Locale/en-US/seeds/nymph_seed.ftl [hunk 9]
```diff
@@ -0,0 +1,3 @@
+ent-NymphSeed = packet of dionea spores
```
Comment (EmoGarbage404) [inline Resources/Locale/en-US/seeds/nymph_seed.ftl:1]: same here.

File: Resources/Prototypes/Entities/Objects/Specific/Hydroponics/nymph_seed.yml [hunk 10]
```diff
@@ -0,0 +1,35 @@
+- type: entity
+  parent: SeedBase
+  name: pack of dionea spores
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Specific/Hydroponics/nymph_seed.yml:3]: calling these "dionae spores" isn't really accurate considering they, besides their appearance, currently have 0 relations to diona besides what other servers implement. Consider just changing it to "nymph seeds/spores"

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
sorry for nitpicking your pr to death, otherwise it looks fine.

**Comment by mirrorcult** [comment]
needs reviews addressed

**Pull request closed without merging**